storage/store.go

```diff
@@ -1164,7 +1166,8 @@ func (s *Store) ExecuteCmd(ctx context.Context, call client.Call) error {
 		// intent independently, so we can't do it in Range.executeCmd.
 		if wiErr, ok := err.(*proto.WriteIntentError); ok {
 			// If inconsistent, return results and resolve asynchronously.
-			if header.ReadConsistency == proto.INCONSISTENT && s.stopper.StartTask() {
+			if header.ReadConsistency == proto.INCONSISTENT &&
+				attempts == 1 && s.stopper.StartTask() {
```

Comment:
I'm probably just not understanding this code, but how does this help alleviate multiple resolve intent goroutines from being created? This code path is returning `retry.Break` which will break us out of the retry loop.